civihr_hrjobcontract_entities/civihr_hrjobcontract_entities.module

```diff
@@ -251,6 +251,43 @@ function civihr_hrjobcontract_entities_init() {
                     WHERE c.deleted = 0
                     ORDER BY t.job_contract_id");
 
+
+        db_query('DROP VIEW IF EXISTS hrjc_contact_details');
+        db_query("CREATE OR REPLACE VIEW hrjc_contact_details AS
+          SELECT c.id AS contact_id,
+            GROUP_CONCAT(DISTINCT CASE WHEN phone_location.name = 'Work' THEN c_phone.phone ELSE '' END
```

Comment:
The idea of the case expression is to filter out phones that are not in Work location or Home location respectively for each column.  If I take it out, for example, if there is someone who has home phone, but not work phone, the home phone will appear in the column for work phone too.  So the case expression replaces phones not in work location for an empty string on work_phone column.